### ISSUE TITLE
Font display problem

### ISSUE THREAD

**Issue by ismanong** [flutter/flutter#121815]
Hello I use google_fonts package and use fonts like Saira, but the fonts will be displayed incomplete. What is the reason for this, please.

The environment is Windows platform.

1. Incomplete display
![image](https://user-images.githubusercontent.com/13862699/222514915-c5a4ba70-fed4-438c-9bc8-3e361e56aabc.png)

2. Occasionally the refresh will be staggered up and down, with the expectation and % on a bottom line again.
![image](https://user-images.githubusercontent.com/13862699/222517641-519d1309-9f7a-4719-9212-163aa8cfcb05.png)

3. Setting the letterSpacing property is also invalid, the gap is too big to look good

**Comment by darshankawar** [comment]
@ismanong 
Can you provide `flutter doctor -v` and minimal code sample that show the reported behavior ?

**Comment by ismanong** [comment]
Hi, I went through the minimal example and found that it was not a flutter issue.
Sorry, it was my own negligence to raise the issue to the community without going through the testing of the minimal example.

All the three issues raised above have been solved, one is the reason of pub package, which has cut on the font; the remaining two issues are caused by my own lack of understanding of the properties of TestSpan.

Thank you for your reminders and replies.

Translated with www.DeepL.com/Translator (free version)
